Plugwise_Smile/Smile.py

```diff
@@ -299,7 +335,6 @@ async def request(
         except etree.XMLSyntaxError:
             _LOGGER.error("Smile returns invalid XML for %s", self._endpoint)
             raise self.InvalidXMLError
-            return None
```

Comment:
I need this None to be returned.
Can we change the code so that it's returned properly?

Reply:
As discussed on discord: let's review tomorrow (either keep the raise or the return but not both). We're probably after what https://github.com/plugwise/Plugwise-Smile/pull/95/commits/561bea42d10fa9f4e56e9933b4607446815070c4 does, right?

Reply:
Yes, correct.